light-zk.js/src/wallet/user.ts

```diff
@@ -577,12 +569,12 @@ export class User {
         `Error in tx.sendTransaction ${e}`,
       );
     }
-    let transactionContainsEncryptedUtxo = false;
-    this.recentTransactionParameters.outputUtxos.map((utxo) => {
-      if (utxo.account.pubkey.toString() === this.account?.pubkey.toString()) {
-        transactionContainsEncryptedUtxo = true;
-      }
-    });
+    // let transactionContainsEncryptedUtxo = false;
+    // this.recentTransactionParameters.outputUtxos.map((utxo) => {
+    //   if (utxo.account.pubkey.toString() === this.account?.pubkey.toString()) {
+    //     transactionContainsEncryptedUtxo = true;
+    //   }
+    // });
```

Comment:
it's not used right?
then it can be removed

Reply:
removed ✅